Source/DafnyCore/Resolver/ModuleResolver.cs

```diff
@@ -1262,6 +1262,15 @@ public void ResolveTopLevelDecls_Core(List<TopLevelDecl> declarations,
         FillInPostConditionsAndBodiesOfPrefixLemmas(declarations);
       }
 
+      // A function is not allowed to be used naked in its own SCC. Also, a function is not allowed to be used
```

Comment:
What does the term naked here mean?

Reply:
It means the function name is used without applying it to arguments. (This nomenclature is not new in this PR.) I clarified in the comment.